src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -191,6 +188,9 @@
     <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.LookupAccountNameW.cs">
       <Link>Common\Interop\Windows\mincore\Interop.LookupAccountNameW.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Windows\mincore\Interop.SHGetKnownFolderPath.cs">
```

Comment:
They both have desktop only APIs. The label isn't right, or at best is misleading.